xls/dslx/bytecode/bytecode_interpreter_test.cc

```diff
@@ -1253,6 +1253,68 @@ fn main() -> MyU9 {
   EXPECT_THAT(value.GetBitValueUint64(), IsOkAndHolds(0x1ff));
 }
 
+TEST(BytecodeInterpreterTest, AssertEqFailProcs) {
+  constexpr std::string_view kProgram = R"(
+#[test_proc]
+proc BTester {
+    terminator: chan<bool> out;
+
+    init { (u32:0) }
+
+    config(terminator: chan<bool> out) {
+        (terminator,)
+    }
+
+    next(tok: token, state: u32) {
+        let count = state;
+        let _ = assert_eq(count, u32:0);
```

Comment:
curious if this is also failing when comparing directly against `state`? (I think it would)

Reply:
I think it does.
Should I change this to reduce number of variables?

Reply:
as you want, let me know what you find more readable.

Reply:
I think I'll remove count and use state directly